model-client/src/commonMain/kotlin/org/modelix/model/client2/IModelClientV2.kt

```diff
@@ -85,4 +85,6 @@ interface IModelClientV2 {
     suspend fun <R> query(branch: BranchReference, body: (IMonoStep<INode>) -> IMonoStep<R>): R
 
     suspend fun <R> query(repositoryId: RepositoryId, versionHash: String, body: (IMonoStep<INode>) -> IMonoStep<R>): R
+
+    suspend fun getObjects(repository: RepositoryId, keys: Sequence<String>): Map<String, String>
```

Comment:
From #723:

Any reason for using a sequence here as the input type when the output type is not lazy?